vertx-web-common/src/main/java/io/vertx/ext/web/codec/sse/SseBodyCodec.java

```diff
@@ -0,0 +1,211 @@
+package io.vertx.ext.web.codec.sse;
+
+import io.vertx.codegen.annotations.Nullable;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.streams.ReadStream;
+import io.vertx.core.streams.WriteStream;
+import io.vertx.ext.web.codec.BodyCodec;
+import io.vertx.ext.web.codec.spi.BodyStream;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * A codec for processing and decoding Server Sent Event from streaming HTTP body content .
+ */
+public class SseBodyCodec implements BodyCodec<Void> {
```

Comment:
This should be a Vert.x API object (annotated with `@VertxGen`)

Otherwise, it won't be available in Vert.x generated bindings (Mutiny, RxJava, Kotlin)